src/LightSpeed/userFunction.js

```diff
@@ -139,11 +147,15 @@ var LSUserFunction = GObject.registerClass({
         this._bindModel(this._global.getModel(this._currentLevel));
     }
 
-    _bindModel(model) {
-        if (this._model)
-            this._model = null;
+    _unbindLocalModel() {
+        this._localModel = null;
+    }
+
+    _bindLocalModel(model) {
+        if (this._localModel)
+            this._localModel = null;
```

Comment:
Why do you set this do `null` if you're already assigning it below? I see the code was already there, but I don't understand why that was needed eitheer.